lib/srv/desktop/rdp/rdpclient/client.go

```diff
@@ -237,12 +237,14 @@ func (c *Client) connect(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 
-	// Addr and username strings only need to be valid for the duration of
+	// These strings only need to be valid for the duration of
 	// C.connect_rdp. They are copied on the Rust side and can be freed here.
```

Comment:
I suppose we could pin them instead without copying into a C allocation, but I don't feel the need to change this now given where v14 is in its lifecycle.

Reply:
Ah, I suppose we're making the C allocation so that we can pass a null terminated string.

Doing this without allocation would require we change the Cgo call to pass both a pointer and a length.